src/coreclr/CMakeLists.txt

```diff
@@ -173,6 +173,10 @@ endif(CLR_CMAKE_HOST_WIN32)
 #----------------------------------
 include(clrdefinitions.cmake)
 
+if(FEATURE_STANDALONE_GC)
```

Comment:
The define is added in the gc/CMakeLists.txt, I am not sure I understand why we would need to add it here.

Reply:
Ah, I see, I have missed that it is not used just inside of the GC code, but also in the runtime to enable the code that loads the standalone GC.